xAPI.md

```diff
@@ -1193,7 +1193,9 @@ Run-Time Environment. See [Appendix C](#AppendixC) for examples of each format.
 	</tr>
 	<tr>
 		<td>numeric</td>
-		<td>A range of numbers represented by a minimum and a maximum delimited by <code>:</code>.
+		<td>A range of numbers represented by a minimum and a maximum delimited by <code>[:]</code>. 
```

Comment:
Works for me, but only if the issue's a blocker for 1.0.3. This (both here and with other SCORM areas) is vague enough right now it is unlikely to be implemented consistently enough such that all LRSs could accept each other's data in likely corner cases.